{
    "cveId": "CVE-2019-20431",
    "version": "1.0.0",
    "timestamp": "2024-12-12T22:08:52.294043+00:00",
    "description": "In the Lustre file system before 2.12.3, the ptlrpc module has an osd_map_remote_to_local out-of-bounds access and panic due to the lack of validation for specific fields of packets sent by a client. osd_bufs_get in the osd_ldiskfs module does not validate a certain length value.",
    "keyphrases": {
        "rootcause": "lack of validation",
        "weakness": "out-of-bounds access",
        "impact": "panic",
        "vector": "",
        "attacker": "client",
        "product": "Lustre file system",
        "version": "before 2.12.3",
        "component": "ptlrpc module"
    }
}
